src/Main.java

```diff
@@ -1,5 +1,19 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Задание 1");
+        String firstName = "Ivan";
+        String middleName = "Ivanovich";
+        String lastName = "Ivanov";
+        String space = " ";
+        String fullName = lastName + space + firstName + space + middleName;
```

Comment:
Можно было не создавать переменную, а сделать ее in-line
```suggestion
        String fullName = lastName + " " + firstName + " " + middleName;
```